FluentUI/src/main/java/com/microsoft/fluentui/contextualcommandbar/CommandItem.kt

```diff
@@ -0,0 +1,15 @@
+package com.microsoft.fluentui.contextualcommandbar
+
+import android.support.annotation.DrawableRes
+
+interface CommandItem {
+
+    @DrawableRes
+    fun getIcon(): Int
```

Comment:
A more general question - I have asked the same on the iOS/Swift implementation PR - it seems there may be scenarios where there is a need to display a command button with a localized string/label rather than an icon. Can we add that functionality here?
I am given to understand that app which hosted the original code this PR is based on does also have that functionality (though I have not checked so am not sure how accurate that is)